### PR TITLE
[HER-105] Create pagination command

### PR DESCRIPTION
**Pull request by ConstantinescuAndrei**
## Description
 Create CreatePagination interface and his implementation

## Related
[HER-105](https://nerds-interns.atlassian.net/jira/software/projects/HER/boards/1?issueParent=10022&selectedIssue=HER-105)

### CODE REVIEW

**Review by TBiannca — DISMISSED** [review]

**Review by Urzica-Platon-Casian — DISMISSED** [review]
Looks good

File: src/Data/Pagination/Common/CreatePagination.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
+namespace Data.Pagination.Common
```
Comment (RaduBuciuceanu) [inline src/Data/Pagination/Common/CreatePagination.cs:1]: Should we move this class under the `Data.Pagination` folder? Should we also make sure to adjust the namespaces?

**Review by RaduBuciuceanu — CHANGES_REQUESTED** [review]

**Review by danidoff — APPROVED** [review]
Nicely done my friend !!!

File: src/Data/Pagination/Common/CreatePagination.cs (hunk 1 above)
Comment (ConstantinescuAndrei) [reply src/Data/Pagination/Common/CreatePagination.cs]: done

**Review by danidoff — APPROVED** [review]

**Review by RaduBuciuceanu — APPROVED** [review]

**Pull request merged**